cli/src/commands/sign.rs

```diff
@@ -0,0 +1,117 @@
+// Copyright 2023 The Jujutsu Authors
```

Comment:
nit: maybe 2024 or 2025? or maybe this work was started in 2023 so it should be 2023?

Reply:
I `jj duplicate`d the commit, so I suppose the work started in 2023.